docs/extend/cli_plugins.md

```diff
@@ -75,6 +75,19 @@ A plugin is required to support all of the global options of the
 top-level CLI, i.e. those listed by `man docker 1` with the exception
 of `-v`.
 
+## Connecting to the docker engine
+
+For consistency plugins should prefer to dial the engine by using the
+`system dial-stdio` subcommand of the main Docker CLI binary.
+
+To facilitate this plugins will be executed with the
+`$DOCKER_CLI_PLUGIN_ORIGINAL_CLI_COMMAND` environment variable
```

Comment:
Given that plugins (when called through the `docker` cli) are executed as child-process of `docker` Would there be an approach to get the parent ID, and get the information from that? 🤔 (something like [`os.Getppid()`](https://golang.org/pkg/os/#Getppid), or do we need more information than the executable itself?

https://www.socketloop.com/tutorials/golang-get-executable-name-behind-process-id-example

Reply:
That seems very platform specific (I don't know about it's portability to MacoS, let alone Windows) and probably quite fragile even on Linux compared with this approach which is pretty simple.